spec/factories/messages.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :message do
+    subject "test subject"
+    status "Sent"
+    body "This is a test message!"
+    sent_at 2.hours.ago
+    association :recipient, factory: :recipient
```

Comment:
I prefer to not have the associations in the factories. I find this slows the tests to much. There are many tests for messages that will not need the recipient or channel, and those that do can handle them. I have found that when the associations _are_ in the factory, it cascades. You asked for a recipient and a channel; but what if their factories have a person association ? And what if Recipient has an association to Notification ? You get the idea, you end up creating 12 objects for one test. And additional problem is that if recipient and channel created a Person, it would not be the same instance.
I am going to merge this to move forward, and create a new issue.